src/main/java/duke/dukeobject/ExpenseList.java

```diff
@@ -3,50 +3,38 @@
 import duke.exception.DukeException;
 import duke.exception.DukeRuntimeException;
 
+import javax.swing.text.View;
```

Comment:
Not sure where this is used; if im missing something point it out, else it can be removed